[2024-04-29T05:38:37.536+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl.get_files scheduled__2024-04-26T00:00:00+00:00 [queued]>
[2024-04-29T05:38:37.756+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl.get_files scheduled__2024-04-26T00:00:00+00:00 [queued]>
[2024-04-29T05:38:37.765+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-04-29T05:38:37.948+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): get_files> on 2024-04-26 00:00:00+00:00
[2024-04-29T05:38:38.071+0000] {standard_task_runner.py:60} INFO - Started process 2395 to run task
[2024-04-29T05:38:38.141+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'etl', 'get_files', 'scheduled__2024-04-26T00:00:00+00:00', '--job-id', '49', '--raw', '--subdir', 'DAGS_FOLDER/etl.py', '--cfg-path', '/tmp/tmpnogfrjm6']
[2024-04-29T05:38:38.160+0000] {standard_task_runner.py:88} INFO - Job 49: Subtask get_files
[2024-04-29T05:38:38.535+0000] {task_command.py:423} INFO - Running <TaskInstance: etl.get_files scheduled__2024-04-26T00:00:00+00:00 [running]> on host 422c09390d6c
[2024-04-29T05:38:38.928+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl' AIRFLOW_CTX_TASK_ID='get_files' AIRFLOW_CTX_EXECUTION_DATE='2024-04-26T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-26T00:00:00+00:00'
[2024-04-29T05:38:38.944+0000] {etl.py:24} INFO - 5 files found in /opt/***/dags/data
[2024-04-29T05:38:38.944+0000] {python.py:202} INFO - Done. Returned value was: ['/opt/***/dags/data/github_events_02.json', '/opt/***/dags/data/github_events_03.json', '/opt/***/dags/data/github_events_04.json', '/opt/***/dags/data/github_events_05.json', '/opt/***/dags/data/github_events_01.json']
[2024-04-29T05:38:39.054+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=etl, task_id=get_files, execution_date=20240426T000000, start_date=20240429T053837, end_date=20240429T053839
[2024-04-29T05:38:39.179+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-04-29T05:38:39.289+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
